eos/README.md

```diff
@@ -0,0 +1,27 @@
+# EOS Docker Scripts
```

Comment:
Doesn't make sense to me, this is for deploying your own network, it's not our usecase.